app/Http/Controllers/TechniqueController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Technique;
+use App\Models\Review;
+use Illuminate\View\View;
+
+class TechniqueController extends Controller
+{
+    public function index(): View
+    {
+        $viewData = [];
+        $viewData['title'] = trans('titles.home');
+        $viewData['selected'] = 'techniques';
+        $viewData['technique'] = Technique::all($columns = ['*']);
+
+        return view('technique.index')->with('viewData', $viewData);
+    }
+
+    public function show(string $id): View
```

Comment:
En el modelo definiste id como int pero en el metodo show del controlador el parametro lo tienes como string
```suggestion
    public function show(int $id): View
```